internal/resource/persistence/schema_setup_job_builder.go

```diff
@@ -33,6 +33,8 @@ import (
 	"sigs.k8s.io/controller-runtime/pkg/controller/controllerutil"
 )
 
+const ServiceNameSuffix = "schema-setup"
```

Comment:
Could you please add some godoc explaining the goal of this variable ? 
It's easy to understand in this PR context but I think it will be harder in few months :D